ksqldb-common/src/main/java/io/confluent/ksql/schema/ksql/LogicalSchema.java

```diff
@@ -123,10 +127,23 @@ public boolean valueContainsAny(final Set<ColumnName> names) {
    *
    * @param windowed indicates that the source is windowed; meaning {@code WINDOWSTART} and {@code
    * WINDOWEND} columns will added to the value schema to represent the window bounds.
+   *
+   * @param pseudoColumnVersion indicates the pseudocolumn version number of the given query.
```

Comment:
nit: the first part of this docstring is not very informative since it simply repeats the name of the parameter. How about something like "version number that determines the set of pseudocolumns to be added to the schema" or even simply "determines the set of pseudocolumns to be added to the schema".